lib/ClangImporter/ImportType.cpp

```diff
@@ -2661,6 +2664,8 @@ ArgumentAttrs ClangImporter::Implementation::inferDefaultArgument(
           return argumentAttrs;
         if (camel_case::sameWordIgnoreFirstCase(word, "action"))
           return argumentAttrs;
+        if (camel_case::sameWordIgnoreFirstCase(word, "event"))
```

Comment:
how does this affect objc interop? Isn't it source breaking even without C++ interop?

Reply:
This code path is only executed with C++ interop enabled. This entire mechanism was added in https://github.com/apple/swift/pull/59421.

Reply:
got it, makes sense.

Reply:
Ah nice, I see I must have missed adding "event" when I did the original. 

I bet this screenshot from the diff at https://github.dev/plotfi/cxx-interop-diff/pull/12 is whats being fixed:

<img width="1471" alt="Untitled" src="https://github.com/apple/swift/assets/34139736/3a1884b6-ada7-4315-bf2f-024baeb7b4c5">


Nice work.